pandas/core/arrays/string_.py

```diff
@@ -282,8 +283,23 @@ def astype(self, dtype, copy=True):
         return super().astype(dtype, copy)
 
     def _reduce(self, name, skipna=True, **kwargs):
+        if name in ["min", "max"]:
+            return getattr(self, name)(skipna=skipna, **kwargs)
+
         raise TypeError(f"Cannot perform reduction '{name}' with string dtype")
 
+    def min(self, axis=None, out=None, keepdims=False, skipna=True):
```

Comment:
Updated, assuming by test with numpy you mean the validation functions in /compat/numpy/function?

Reply:
Yes, indeed, those validation functions can be used to check additional keywords. 

In addition, we should also test this in the tests (so test that np.min(a) works, because that is the only reason we would add those keywords)